department-of-veterans-affairs/vets-website
# Increase list loop hint contrast

## Summary

Fix accessibility issue, hint text has insufficient contrast to background in the list loop blocks. Slightly lightens the list loop "add another" and review blocks throughout Pensions. 

## How to run in local environment
1. Check out this branch locally
2. In your local environment, set the pension_form_enabled flipper to 'enabled' at http://localhost:3000/flipper/features/pension_form_enabled
3. Run vets-website and vets-api

## How to verify
1. `/financial/income-sources`, `household/marital-status/spouse-marriages`, and `/financial/medical-expenses/add`
2. Fill in all required fields and click 'add another'
3. Confirm that the hint text has sufficient contrast

## Related issue(s)

- [Link to Ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/73305)
- [Link to Epic](https://github.com/department-of-veterans-affairs/va.gov-team/issues/63349)

## Testing done

- Reviewed with axe devtools

## Screenshots

|         | Before | After |
| ------- | ------ | ----- |
| Address |   <img width="1433" alt="Screenshot 2024-01-12 at 11 46 29 AM" src="https://github.com/department-of-veterans-affairs/vets-website/assets/21046714/0091cb78-36d5-45ed-9a15-7e8541983c30">    |    <img width="1427" alt="Screenshot 2024-01-12 at 11 38 29 AM" src="https://github.com/department-of-veterans-affairs/vets-website/assets/21046714/8e4d7832-4a8b-4bdd-a03b-10649fe8cfbf">  |

## What areas of the site does it impact?

Pension Benefits

## Acceptance criteria
- [Link to Ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/73305)

### Quality Assurance & Testing


### Error Handling


### Authentication
